src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SignedXml.cs

```diff
@@ -796,49 +796,19 @@ private static IList<string> DefaultSafeTransformMethods
         // HKLM\Software\Microsoft\.NETFramework\Security\SafeCanonicalizationMethods
         // key.  Each REG_SZ entry in this key will be considered a canonicalziation algorithm URI that should be
         // allowed by SignedXml instances on this machine.
-        [RegistryPermission(SecurityAction.Assert, Unrestricted = true)]
         private static List<string> ReadAdditionalSafeCanonicalizationMethods()
```

Comment:
Next is
* Delete this method
* Find the callers.
* Determine what they would do if an empty list is returned, delete the other codepaths.